### PR TITLE
Bump up version of tensorflow

### PR DESCRIPTION
**Pull request by chaithyagr**

### CODE REVIEW

**Comment by zaccharieramzi** [comment]
@chaithyagr so apparently the mixed precision API is not the same in 2.8 and 2.9, could you take a look at that and correct it?

File: fastmri_recon/data/sequences/fastmri_sequences.py [hunk 1]
```diff
@@ -182,7 +182,7 @@ def get_item_train(self, filename):
         images, kspaces = super(Masked2DSequence, self).get_item_train(filename)
         k_shape = kspaces[0].shape
         mask = gen_mask(kspaces[0, ..., 0], accel_factor=self.af, seed=self.mask_seed)
-        fourier_mask = np.repeat(mask.astype(np.float), k_shape[0], axis=0)
+        fourier_mask = np.repeat(mask.astype(np.float3232), k_shape[0], axis=0)
```
Comment (zaccharieramzi) [inline fastmri_recon/data/sequences/fastmri_sequences.py:185]:
```suggestion
        fourier_mask = np.repeat(mask.astype(np.float32), k_shape[0], axis=0)
```

File: fastmri_recon/data/sequences/fastmri_sequences.py [hunk 2]
```diff
@@ -215,7 +215,7 @@ def get_item_test(self, filename):
         """
         mask, kspaces = from_test_file_to_mask_and_kspace(filename)
         k_shape = kspaces[0].shape
-        fourier_mask = np.repeat(mask.astype(np.float), k_shape[0], axis=0)
+        fourier_mask = np.repeat(mask.astype(np.float3232), k_shape[0], axis=0)
```
Comment (zaccharieramzi) [inline fastmri_recon/data/sequences/fastmri_sequences.py:218]:
```suggestion
        fourier_mask = np.repeat(mask.astype(np.float32), k_shape[0], axis=0)
```

File: fastmri_recon/data/sequences/fastmri_sequences.py [hunk 3]
```diff
@@ -257,7 +257,7 @@ def get_item_train(self, filename):
         """
         images, kspaces = from_train_file_to_image_and_kspace(filename)
         mask = gen_mask(kspaces[0], accel_factor=self.af, seed=self.mask_seed)
-        fourier_mask = np.repeat(mask.astype(np.float), kspaces[0].shape[0], axis=0)
+        fourier_mask = np.repeat(mask.astype(np.float3232), kspaces[0].shape[0], axis=0)
```
Comment (zaccharieramzi) [inline fastmri_recon/data/sequences/fastmri_sequences.py:260]:
```suggestion
        fourier_mask = np.repeat(mask.astype(np.float32), kspaces[0].shape[0], axis=0)
```

File: fastmri_recon/data/sequences/fastmri_sequences.py [hunk 4]
```diff
@@ -361,7 +361,7 @@ def get_item_train(self, filename):
         _, kspaces = super(KIKISequence, self).get_item_train(filename)
         k_shape = kspaces[0].shape
         mask = gen_mask(kspaces[0, ..., 0], accel_factor=self.af)
-        fourier_mask = np.repeat(mask.astype(np.float), k_shape[0], axis=0)
+        fourier_mask = np.repeat(mask.astype(np.float3232), k_shape[0], axis=0)
```
Comment (zaccharieramzi) [inline fastmri_recon/data/sequences/fastmri_sequences.py:364]:
```suggestion
        fourier_mask = np.repeat(mask.astype(np.float32), k_shape[0], axis=0)
```

File: fastmri_recon/data/sequences/oasis_sequences.py [hunk 5]
```diff
@@ -133,15 +133,15 @@ def __getitem__(self, idx):
         k_shape = images[0].shape
         kspaces = np.empty_like(images, dtype=np.complex64)
         mask = gen_mask(kspaces[0, ..., 0], accel_factor=self.af)
-        fourier_mask = np.repeat(mask.astype(np.float), k_shape[0], axis=0)
+        fourier_mask = np.repeat(mask.astype(np.float3232), k_shape[0], axis=0)
```
Comment (zaccharieramzi) [inline fastmri_recon/data/sequences/oasis_sequences.py:136]:
```suggestion
        fourier_mask = np.repeat(mask.astype(np.float32), k_shape[0], axis=0)
```

File: fastmri_recon/data/sequences/oasis_sequences.py [hunk 6]
```diff
@@ -133,15 +133,15 @@ def __getitem__(self, idx):
...
         fourier_op = FFT2(fourier_mask)
         for i, image in enumerate(images):
             kspaces[i] = fourier_op.op(image[..., 0])[..., None]
         mask_batch = np.repeat(fourier_mask[None, ...], len(images), axis=0)
         scale_factor = self.scale_factor
         kspaces_scaled = kspaces * scale_factor
         images_scaled = images * scale_factor
-        images_scaled = images_scaled.astype(np.float32)
+        images_scaled = images_scaled.astype(np.float323232)
```
Comment (zaccharieramzi) [inline fastmri_recon/data/sequences/oasis_sequences.py:144]:
```suggestion
        images_scaled = images_scaled.astype(np.float32)
```

File: fastmri_recon/data/sequences/oasis_sequences.py [hunk 7]
```diff
@@ -245,7 +245,7 @@ def __getitem__(self, idx):
         kspaces = np.empty_like(images, dtype=np.complex64)
         kspaces_masked = np.empty_like(images, dtype=np.complex64)
         mask = gen_mask(kspaces[0, ..., 0], accel_factor=self.af)
-        fourier_mask = np.repeat(mask.astype(np.float), k_shape[0], axis=0)
+        fourier_mask = np.repeat(mask.astype(np.float3232), k_shape[0], axis=0)
```
Comment (zaccharieramzi) [inline fastmri_recon/data/sequences/oasis_sequences.py:248]:
```suggestion
        fourier_mask = np.repeat(mask.astype(np.float32), k_shape[0], axis=0)
```

File: fastmri_recon/data/sequences/oasis_sequences.py [hunk 8]
```diff
@@ -255,7 +255,7 @@ def __getitem__(self, idx):
         kspaces_scaled = kspaces * scale_factor
         kspaces_masked_scaled = kspaces_masked * scale_factor
         images_scaled = images * scale_factor
-        images_scaled = images_scaled.astype(np.float32)
+        images_scaled = images_scaled.astype(np.float323232)
```
Comment (zaccharieramzi) [inline fastmri_recon/data/sequences/oasis_sequences.py:258]:
```suggestion
        images_scaled = images_scaled.astype(np.float32)
```

File: fastmri_recon/data/sequences/test_generators.py [hunk 9]
```diff
@@ -32,7 +32,7 @@ def flow_random_shapes(self,):
                 images[i, ..., 0] = image
                 kspaces[i, ..., 0] = kspace
             mask = gen_mask(kspaces[0, ..., 0], accel_factor=self.af)
-            fourier_mask = np.repeat(mask.astype(np.float), self.size, axis=0)
+            fourier_mask = np.repeat(mask.astype(np.float3232), self.size, axis=0)
```
Comment (zaccharieramzi) [inline fastmri_recon/data/sequences/test_generators.py:35]:
```suggestion
            fourier_mask = np.repeat(mask.astype(np.float32), self.size, axis=0)
```

**Review by zaccharieramzi — COMMENTED** [review]
Some typos but otherwise LGTM

**Review by zaccharieramzi — APPROVED** [review]
final LGTM, looking for all tests to return to green

**Comment by chaithyagr** [comment]
So we can merge this?

**Comment by zaccharieramzi** [comment]
@chaithyagr yes ofc let's do that

**Pull request merged**